<launch>

   <include file="$(find controller_tutorial)/launch/controller_test.launch">
      <arg name="multi_robot"          value="/robot_1"/>
      <arg name="path_csvfile"         value="$(find controller_tutorial)/path/straight.csv"/>
      <arg name="use_rviz"             value="false"/>
   </include>

   <include file="$(find controller_tutorial)/launch/controller_test.launch">
      <arg name="multi_robot"          value="/robot_2"/>
      <arg name="path_csvfile"         value="$(find controller_tutorial)/path/straight.csv"/>
      <arg name="use_rviz"             value="false"/>
   </include>

   <include file="$(find controller_tutorial)/launch/controller_test.launch">
      <arg name="multi_robot"          value="/robot_3"/>
      <arg name="path_csvfile"         value="$(find controller_tutorial)/path/straight.csv"/>
      <arg name="use_rviz"             value="false"/>
   </include>

</launch>
